hal/network/lwip/cellular/pppncpnetif.cpp

```diff
@@ -151,28 +178,26 @@ void PppNcpNetif::loop(void* arg) {
 
 int PppNcpNetif::up() {
     NetifEvent ev = NetifEvent::Up;
-    int r = os_queue_put(queue_, &ev, 0, nullptr);
```

Comment:
Again, we've just removed the workaround for cases where the queue is filled up and the event will be completely missed.